src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -730,7 +730,9 @@ class FlowCrudSpec extends HealthCheckSpecification {
         def newIsl = islUtils.replug(isl, false, notConnectedIsl, true, true)
 
         islUtils.waitForIslStatus([isl, isl.reversed], MOVED)
-        islUtils.waitForIslStatus([newIsl, newIsl.reversed], DISCOVERED)
+        Wrappers.wait(discoveryFailedInterval + WAIT_OFFSET) {
+            islUtils.waitForIslStatus([newIsl, newIsl.reversed], DISCOVERED)
```

Comment:
Wait inside wait? I don't really get it. Either change how `waitForIslStatus` works or remove it and do a common wait for isls without `islUtils.waitForIslStatus`